src/pages/hydra.tsx

```diff
@@ -74,8 +75,31 @@ const IndexPage = () => {
         }
         image={<img loading="lazy" alt="" src={hydraProcess} />}
       />
+
+<FeatureImage
+        overline={'A proven solution'}
+        title={<>Millions of users? No problem</>}
+        description={
+          <>
+            If you need a reliable open source solution Hydra is the right tool for the job.
+            <br></br>Watch the video to learn how Hydra powers Sainsbury's My ID, handling hundreds of users per second with ease.
+          </>
```

Comment:
Please no `<br>` tags, keep it all in one line and let the layout engine decide how to break the text.

```suggestion
            If you need a reliable open source solution Hydra is the right tool for the job. Watch the video to learn how Hydra powers Sainsbury's My ID, handling hundreds of users per second with ease.
          </>
```